alternator/serialization.cc

```diff
@@ -245,6 +245,25 @@ rjson::value deserialize_item(bytes_view bv) {
     return deserialized;
 }
 
+// If the value in bv (supposedly created earlier by serialize_item()) has the
```

Comment:
I'm lost, what's `bv`? Perhaps you meant `mv`?
Also, on the same note, what are "(S, B or N)" from the commit's msg? On a related note, in the commit message, perhaps rewording "which takes value stored serialized" to "which takes serialized value stored" read better.

Reply:
by is the "bytes view". S B and N are string, blob and number respectively, are the types that a DynamoDB key can have (DynamoDB has other types, like map, list, etc., but those can't be a key). I'll work on improving those comments and commit messages if they aren't clear enough.

Reply:
A pro tip that @bhalevy gave me is to use both the full and short/abbreviated name when introducing an abbreviation, so here it'd be something like:
> serialized types String (S), Blob (B) and Number (N)

But this is only good when you use the abbreviated name later, so as not the repeat the full name again and make the text shorter. Here it's of no use, really, because you use it only once, but maybe you can use it in the future.